src/python/bot/tasks/minimize_task.py

```diff
@@ -1085,14 +1087,35 @@ def _run_libfuzzer_testcase(testcase, testcase_file_path):
   return CrashResult(return_code, crash_time, output)
 
 
+def run_libfuzzer_engine(tool_name, target_name, arguments, testcase_path,
+                         output_path, timeout):
+  """Run the libFuzzer engine."""
+  if environment.is_trusted_host():
+    from bot.untrusted_runner import tasks_host
+    # TODO(ochang): Remove hardcode.
+    return tasks_host.process_testcase('libFuzzer', tool_name, target_name,
+                                       arguments, testcase_path, output_path,
+                                       timeout)
+
+  target_path = engine_common.find_fuzzer_path(
+      environment.get_value('BUILD_DIR'), target_name)
+
```

Comment:
What if target_path is not found, should we account for that ?

Reply:
added a check here, to return a dummy value.